File: src/main/java/models/Reunion.java
package models;

import java.time.LocalDateTime;

public class Reunion {
    private int idReunion;
    private int idRH;
    private int idCandidat;
    private LocalDateTime date;
    private String link;
    private String nomCandidat;

    public Reunion(int idReunion, int idRH, int idCandidat, LocalDateTime date, String link) {
        this.idReunion = idReunion;
        this.idRH = idRH;
        this.idCandidat = idCandidat;
        this.date = date;
        this.link = link;
    }

    public Reunion(int idRH, int idCandidat, LocalDateTime date, String link) {
        this.idRH = idRH;
        this.idCandidat = idCandidat;
        this.date = date;
        this.link = link;
    }

    public Reunion() {
    }

    public int getIdReunion() {
        return idReunion;
    }

    public void setIdReunion(int idReunion) {
        this.idReunion = idReunion;
    }

    public int getIdRH() {
        return idRH;
    }

    public void setIdRH(int idRH) {
        this.idRH = idRH;
    }

    public int getIdCandidat() {
        return idCandidat;
    }

    public void setIdCandidat(int idCandidat) {
        this.idCandidat = idCandidat;
    }

    public LocalDateTime getDate() {
        return date;
    }

    public void setDate(LocalDateTime date) {
        this.date = date;
    }

    public String getLink() {
        return link;
    }

    public void setLink(String link) {
        this.link = link;
    }

    @Override
    public String toString() {
        return "Reunion{" +
                "idReunion=" + idReunion +
                ", idRH=" + idRH +
                ", idCandidat=" + idCandidat +
                ", date=" + date +
                ", link='" + link + '\'' +
                '}';
    }

    public String getNomCandidat() {
        return nomCandidat;
    }

    public void setNomCandidat(String nomCandidat) {
        this.nomCandidat = nomCandidat;
    }
}

File: symfony-migration/src/Entity/Reunion.php
<?php

namespace App\Entity;

use App\Repository\ReunionRepository;
use Doctrine\ORM\Mapping as ORM;

#[ORM\Entity(repositoryClass: ReunionRepository::class)]
#[ORM\Table(name: 'reunion')]
class Reunion
{
    #[ORM\Id]
    #[ORM\GeneratedValue]
    #[ORM\Column(name: 'idReunion')]
    private ?int $id = null;

    #[ORM\Column(name: 'idRH')]
    private int $idRh = 0;

    #[ORM\ManyToOne]
    #[ORM\JoinColumn(name: 'idCandidat', referencedColumnName: 'idCandidat', nullable: false)]
    private ?Candidat $candidat = null;

    #[ORM\Column(name: 'date', type: 'datetime_immutable')]
    private ?\DateTimeImmutable $date = null;

    #[ORM\Column(name: 'link', length: 255)]
    private string $link = '';

    public function getId(): ?int { return $this->id; }
    public function getIdRh(): int { return $this->idRh; }
    public function setIdRh(int $idRh): self { $this->idRh = $idRh; return $this; }
    public function getCandidat(): ?Candidat { return $this->candidat; }
    public function setCandidat(?Candidat $candidat): self { $this->candidat = $candidat; return $this; }
    public function getDate(): ?\DateTimeImmutable { return $this->date; }
    public function setDate(?\DateTimeImmutable $date): self { $this->date = $date; return $this; }
    public function getLink(): string { return $this->link; }
    public function setLink(string $link): self { $this->link = $link; return $this; }
}
